### DISCUSSION TITLE
Fixed Percent and Time Remaining on Print Finish Compiling Error

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#432] [Q&A] [answered]
Compiling the lasted update for manual mesh 3X3 I get this error. Fixed Percent and Time Remaining on Print Finish
![Compile](https://user-images.githubusercontent.com/76499230/114231973-2b1e4b00-9949-11eb-9b4b-807c916c05b1.jpg)

**Comment by Jyers** [comment, marked as answer]
This issue has been fixed in the latest commit
